common/static/js/vendor/jquery.scrollTo.min.js

```diff
@@ -0,0 +1 @@
+../../../../node_modules/jquery.scrollto/jquery.scrollTo.min.js
```

Comment:
@bjacobel This was from before you took this over, but we are no longer using symlinks to bring in vendor libraries. This should follow the pattern used by the jquery library: add the unminified file to the list of vendor libraries in assets.py, and then change the dependencies to refer to it in the new location.